src/Workspaces/CSharp/Portable/Formatting/Rules/SpacingFormattingRule.cs

```diff
@@ -26,8 +26,9 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
             var previousParentKind = previousToken.Parent.Kind();
             var currentParentKind = currentToken.Parent.Kind();
 
-            // For Method Declaration
-            if (currentToken.IsOpenParenInParameterList() && previousKind == SyntaxKind.IdentifierToken)
+            // For Method Declaration and Conversion Operator Declaration with Predefined Return types
+            if (currentToken.IsOpenParenInParameterList() &&
+                (previousKind == SyntaxKind.IdentifierToken || (previousToken.IsKeyword() && currentToken.Parent.IsParentKind(SyntaxKind.ConversionOperatorDeclaration))))
```

Comment:
The regular operators are handled [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/Formatting/Rules/TokenBasedFormattingRule.cs,203). I am looking if they have any supporting testcases(and hope there will be)